src/base_object.h

```diff
@@ -36,6 +36,8 @@ class BaseObjectPtrImpl;
 
 class BaseObject : public MemoryRetainer {
  public:
+  enum InternalFields : int { kSlot, kInternalFieldCount };
```

Comment:
```suggestion
  enum InternalFields { kSlot, kInternalFieldCount };
```
Or declare them `int` everywhere, either is fine as long as it's consistent.